src/asl.ts

```diff
@@ -479,21 +514,15 @@ export class ASL {
         },
         ResultPath: FUNCTIONLESS_CONTEXT_JSON_PATH,
         OutputPath: "$",
-        Next: start,
+        Next: ASLGraph.DeferNext,
```

Comment:
Could we use `undefined` as `ASLGraph.DeferNext` if `End: undefined`?

Reply:
the logic is (`Next` or `Default` for `Choice`)
* implicit - `Next === undefined && End === undefined`
* explicit - `Next = ASLGraph.DeferNext`

I've been trying to use the explicit case where I can, but some of the states just have `Next: undefined`.